project_setup/scripts/skeleton

```diff
@@ -71,8 +71,8 @@ def clone_repo(parent_repo, new_repo, org, dir=None):
     )
     run("git tag -d $(git tag -l)", f"Delete all local git tags from {parent_repo}")
     run(
-        fr"find . \( ! -regex '.*/\.git/.*' \) -type f -exec "
-        fr"perl -pi -e s/{parent_repo}/{new_repo}/g {{}} \;",
+        rf"find . \( ! -regex '.*/\.git/.*' \) -type f -exec "
+        rf"perl -pi -e s/{parent_repo}/{new_repo}/g {{}} \;",
```

Comment:
Any reason why you prefer `rf` over `fr`?  I'm not aware of any difference between them.

Reply:
That was a `black` change per [the Action logs](https://github.com/cisagov/development-guide/runs/5336595561?check_suite_focus=true#step:16:49).

Reply:
Well if 'black' says it, it must be true!  Carry on! 👍